src/tasks/authenticate/taskauthenticate.h

```diff
@@ -32,6 +34,7 @@ class TaskAuthenticate final : public Task {
   AuthenticationListener* m_authenticationListener = nullptr;
   AuthenticationListener::AuthenticationType m_authenticationType =
       AuthenticationListener::AuthenticationInBrowser;
+  QUuid metricUuid = QUuid::createUuid();
```

Comment:
Hungarian notation might be oldschool but: `m_metricUuid` would be neat :)

Reply:
Good call, updated